pyglmnet/pyglmnet.py

```diff
@@ -14,10 +15,11 @@ def _lmb(distr, beta0, beta, X, eta):
     return _qu(distr, z, eta)
 
 
-def _qu(distr, z, eta):
+def _qu(distr, z, eta, return_grad=False):
     """The non-linearity."""
-    if distr == 'softplus':
```

Comment:
'softplus' is not really a distribution, no? Maybe we can say `'poisson-softplus'` and `'poisson-linearized'` to be more explicit